docs/jade-getting-started.md

```diff
@@ -271,24 +247,19 @@ Create a "tdr-dev" managed application:
 you must log in as in order to create a TDR billing profile***. It should be a gmail account.
 * Hit create!
 
-## 12. Setup Environment Variable
+## 11. Setup Environment Variable
 There are several ways to go about this, but here is one way that works. You can set up a Z-shell
 configuration to keep your system environment variables. If you don't already have one created, you
-can create one by running `touch ~/.zshrc`. Then, you can open the file in a text editor with `open ~/.zshrc`.
-Below you'll find a list of environment variables needed to run TDR and tests locally. When you run
-`./render-configs.sh`, it populates key and txt files with secrets from vault and environment-specific
-values. On daily setup, you'll need to run the following two commands *in order*:
-```
-./render-configs.sh
-source ~/.zshrc
-```
-An alternate is to run `./render-configs.sh -i` which will put the variables into your clipboard. You
+can create one by running `touch ~/.zshrc`. Then, you can open the file in a text editor with `open ~/.zshrc`. When you run
+`./scripts/render_configs.sh`, it populates key and txt files with secrets from vault and environment-specific
+values. If you are using the setup script, `./scripts/render_configs.sh` should automatically run.
+An alternate is to run `./scripts/render_configs.sh -i` which will put the variables into your clipboard. You
 can then paste these values into an intellij bootRun or test run profile.
 
 ### Environment Variables
 
 While not exhaustive, here's a list that notes the important environment variables to set when running
-`jade-data-repo` locally. These variables override settings in jade-data-repo/application.properties.
+`jade-data-repo` locally that are not set by `./scripts/render_configs.sh`. These variables override settings in jade-data-repo/application.properties.
 You can convert any application.property to an environment variable by switching to upper case and
 every "." to "_".
 
```

Comment:
following line references ZZ but this is not used in this step. could be for a later clean up ticket though.

Reply:
Yeah, I'd say lets follow up on that, when we get to a more comprehensive readme update